docs/UserGuide.md

```diff
@@ -80,31 +81,34 @@ Format: `help`
 
 ### Exiting the program: `exit`
 
-Exits the program.
+After you finish using `ZooKeep`, you can use this command to close it.
 
 Format: `exit`
 
 ---
 
 ### Listing all animals: `list`
 
-Lists all animals under the care of the user.
+This command is convenient in cases where you need to quickly reference all the animals stored in `ZooKeep` or if you
```

Comment:
Perhaps you can say 'stored in your ZooKeep book' instead to standardise with what you wrote below.